framework/output/RunOutputGroup.kt

```diff
@@ -204,13 +204,15 @@ internal class RunOutputGroup constructor(
 
     private suspend fun consumeConceptMapStreamResponse(response: Response.Stream.ConceptMaps) {
         val notificationMgr = StudioState.notification
+        val preferenceMgr = StudioState.preference
         // TODO: enable configuration of displaying GraphOutput for INSERT and UPDATE
         val table = if (response.source != MATCH) null else TableOutput(
             transaction = runner.transactionState, number = tableCount.incrementAndGet()
         ) // TODO: .also { outputs.add(it) }
         val graph = if (response.source != MATCH) null else GraphOutput(
-            transactionState = runner.transactionState, number = graphCount.incrementAndGet()
-        ).also { outputs.add(it); activate(it) }
+                transactionState = runner.transactionState, number = graphCount.incrementAndGet()
+        ).also { if (preferenceMgr.graphOutputEnabled) {outputs.add(it); activate(it)} }
```

Comment:
See my initial comment on this: https://github.com/vaticle/typedb-studio/pull/623#discussion_r965161376

I will try doing a review rather than adding single comments next time and see if this makes them visible to you.

Reply:
Done.